src/e2e/scala/temple/DSL/ParserE2ETest.scala

```diff
@@ -41,20 +41,22 @@ class ParserE2ETest extends FlatSpec with Matchers with DSLParserMatchers {
             Omit(Set(Endpoint.Delete)),
             Readable.All,
             Writable.This,
-            ServiceAuth("username"),
-            Uses(Seq("Booking", "Events")),
+            ServiceAuth.Email,
+            Uses(Seq("Booking", "Event")),
           ),
           structs = Map(
             "Fred" -> StructBlock(
               Map(
                 "field"  -> Attribute(StringType(), valueAnnotations = Set(Nullable)),
-                "friend" -> Attribute(ForeignKey("User")),
+                "friend" -> Attribute(ForeignKey("TempleUser")),
                 "image"  -> Attribute(BlobType(size = Some(10_000_000))),
               ),
               Seq(ServiceEnumerable(byThis = true), Readable.This),
             ),
           ),
         ),
+        "Booking" -> ServiceBlock(Map()),
+        "Event"   -> ServiceBlock(Map()),
```

Comment:
Not that this is in scope for this PR, but I'm curious what would happen in all our generation if no fields were provided by the user